.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,74 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Publish Release
+
+on:
+  push:
+    branches:
+      - "!**"
+    tags:
+    # Trigger workflow when a tag whose name matches the pattern
+    # "apache-arrow-{MAJOR}.{MINOR}.{PATCH}" is pushed.
+      - "*"
```

Comment:
Please specify a more precise pattern here as we do use other tags (e.g. the go one)

Reply:
Oh I forgot to update the pattern before making the PR. Sorry about that. I'll update the pattern asap.